RetsTalk/RetsTalk/Retrospect/Model/RetrospectManager.swift

```diff
@@ -9,41 +9,124 @@ import Foundation
 import Combine
 
 final class RetrospectManager: RetrospectManageable {
-    private(set) var retrospects: [Retrospect] = []
-    private(set) var retrospectsSubject: CurrentValueSubject<[Retrospect], Never>
     private let userID: UUID
+    private var retrospects: [Retrospect] {
+        didSet { retrospectsSubject.send(retrospects) }
+    }
+    private(set) var retrospectsSubject: CurrentValueSubject<[Retrospect], Never>
+    private let retrospectStorage: Persistable
+    private let assistantMessageProvider: AssistantMessageProvidable
     
-    init(userID: UUID) {
+    init(
+        userID: UUID,
+        retrospectStorage: Persistable,
+        assistantMessageProvider: AssistantMessageProvidable
+    ) {
         self.userID = userID
+        self.retrospects = []
         self.retrospectsSubject = CurrentValueSubject(retrospects)
+        self.retrospectStorage = retrospectStorage
+        self.assistantMessageProvider = assistantMessageProvider
     }
     
-    func fetchRetrospects(offset: Int, amount: Int) {
+    func fetchRetrospects(offset: Int, amount: Int) async throws {
+        let finishedRecentRequest = finishedRecentRetrospectFetchRequest(offset: offset, amount: amount)
+        let finishedRecentEntities = try await retrospectStorage.fetch(by: finishedRecentRequest)
+        retrospects.append(contentsOf: finishedRecentEntities)
+    }
+    
+    func fetchinitRetrospects(offset: Int, amount: Int) async throws {
+        let isPinnedRequest = isPinnedRetrospectFetchRequest()
+        let isPinnedEntities = try await retrospectStorage.fetch(by: isPinnedRequest)
+        
+        let isProgressRequest = isProgressRetrospectFetchRequest()
+        let isProgressEntities = try await retrospectStorage.fetch(by: isProgressRequest)
+        
+        let finishedRecentRequest = finishedRecentRetrospectFetchRequest(offset: offset, amount: amount)
+        let finishedRecentEntities = try await retrospectStorage.fetch(by: finishedRecentRequest)
         
+        let resultRetrospects = [isPinnedEntities, isProgressEntities, finishedRecentEntities]
+            .flatMap { $0 }
```

Comment:
```suggestion
        let resultRetrospects = isPinnedEntities + isProgressEntities + finishedRecentEntities
```
네이밍 관련해서 isPinnedEntities와 같이 질문형이 되면 Bool타입처럼 보입니다.

pinnedEntities, inProgressEntities가 더 좋은 이름입니다.